osm2gtfs.py

```diff
@@ -31,18 +27,32 @@
 
 
 def main():
+
+    # Load config json file
+    try:
+        config = json.load(args.config)
```

Comment:
Argparser does this for us. Just not in the group, which did make sense at all. Now it should be fine.